app/models/polling/cert_status_updater.rb

```diff
@@ -1,7 +1,9 @@
 class CertStatusUpdater
   def update_hub_usage_status_for_cert(hub_config_api, certificate_to_check)
-    outcomes = entity_ids_for(certificate_to_check).map { |entity_id| cert_is_in_use_for_entity_id?(hub_config_api, entity_id, certificate_to_check) }
+    # TODO: code below prevents sending multiple emails to removed as BAU
+    return certificate_in_use_event(certificate_to_check) unless Certificate.find_by(id: certificate_to_check.id, in_use_at: nil)
```

Comment:
just looking at this... are you sure it doesn't trigger multiple `CertificateInUseEvents`? ... if we're saying the `in_use_at` has already a value, it means that the `CertificateInUseEvent` has already been called at some point.

Reply:
Yes this is exactly what the problem is. So this is a why I have this hacky workaround at the moment.

Reply:
but is this working around? It'll trigger the event again. We don't want that... it should just return, not return an another event.

Reply:
oh I just spotted, it's `new` not `create` ..